etkdocking/dnd.py

```diff
@@ -20,12 +20,13 @@
 
 
 from __future__ import absolute_import
+from builtins import object
```

Comment:
Why are you explicitly importing from the `builtins` module?

Reply:
Futurize backported 18 builtins from Python 3 to Python 2. If someone is running with Py3, it has no effect, but with Py2 it pulls in the Py3 backported versions. This is actually one of the strengths of this approach that it provides this extra "future" module with these builtins.

The other main option is called python-modernize which converts Py2 code into a common subset of Python 2 and 3, with six as a run-time dependency, futurize converts Py2 code into (almost) standard Py3 code, with future as a run-time dependency.

Because future provides more backported Py3 behaviours, the code resulting from futurize is more likely to work identically on both Py3 and Py2 with less additional manual porting effort. The disadvantages is that well I guess we pollute the namespace slightly, and that future only support Python 2.6, 2.7, and 3.3+. Six supports more backwards compatibility, but 2.6 is already end of life, and 2.7 will be [soon](https://pythonclock.org/).

I tried python-modernize first, and I think python-future is really a nice way to go.

For more information, here is [the python-future explanation](http://python-future.org/imports.html#imports-of-builtins).